scripts/taconstants.py

```diff
@@ -0,0 +1,20 @@
+from typing import NewType, Iterable, Set, Dict, Optional
+
+TAConstants = NewType('TAConstants', Dict[int, Set[str]])
+
+def read_taconstants(fname: str) -> TAConstants:
```

Comment:
Do you expect to reuse this functionality from other scripts?
I know parse.py is already huge, but personally it doesn't bother me.
However, if we do want to make the size of source files more manageable, then I'd prefer to put it in it's own directory so that it's not mixed with other standalone scripts.

Reply:
Potentially. Wasn't sure whether to put it in parse.py or not tbh. Probably could just be added to that script rather than moving it to a different directory.